newbank/server/NewBank.java

```diff
@@ -46,6 +48,7 @@ public synchronized String processRequest(CustomerID customer, String request) {
 		if(customers.containsKey(customer.getKey())) {
 			switch(request) {
 			case "SHOWMYACCOUNTS" : return showMyAccounts(customer);
+			case "ADDNEWACCOUNT" :  return addNewAccount(customer);
```

Comment:
I think in the protocol the string to enter to create a new account is simply NEWACCOUNT - not sure how closely we need to stick to this protocol, potentially it would be helpful to list the currently available options to the user when we ask what they want to do?